addons/mail/__manifest__.py

```diff
@@ -59,7 +59,7 @@
 (technically: Server Actions) to be triggered for each incoming mail.
     """,
     'website': 'https://www.odoo.com/app/discuss',
-    'depends': ['base', 'base_setup', 'bus', 'web_tour'],
+    'depends': ['base', 'base_setup', 'bus', 'web_tour', 'web_editor'],
```

Comment:
Pretty sure we don't want mutual dependencies between `mail` or `web_editor`.
Ideally bridge module, but if not then I would much prefer to just define the `/emoji` command in the module `knowledge`, as it's essentially useful in this module. It's not a big deal that it's not available without knowledge, at least for `16.0` (we could define this bridge module in `master`)

Reply:
I moved the emoji code to knowledge in https://github.com/odoo/enterprise/pull/32252